config/environments/production.rb

```diff
@@ -1,3 +1,5 @@
+require "active_support/core_ext/integer/time"
+
 Rails.application.configure do
```

Comment:
Make sure to doublecheck this file after #1506 is merged, since needed edits are made there